docker/jenkins/Dockerfile.centos7-i386

```diff
@@ -111,6 +111,12 @@ RUN wget http://dl.fedoraproject.org/pub/epel/7/SRPMS/f/fakeroot-1.18.4-2.el7.sr
 # remove users/groups with colliding jenkins ids
 RUN userdel systemd-bus-proxy && userdel systemd-network && groupdel input
 
+# install Qt SDK
```

Comment:
that's a good question -- will consider this as part of the next round of Qt changes (which will unfortunately involve different Qt releases based on Linux flavor)